packages/perseus/src/types.ts

```diff
@@ -673,7 +673,8 @@ export type WidgetProps<
     onFocus: (blurPath: FocusPath) => void;
     onBlur: (blurPath: FocusPath) => void;
     findWidgets: (criterion: FilterCriterion) => ReadonlyArray<Widget>;
-    reviewModeRubric: Rubric;
+    reviewModeRubric?: Rubric | null | undefined;
+    reviewMode: boolean;
```

Comment:
As far as I could tell, the only widget that was really using the contents of `reviewModeRubric` was Radio. The other widgets were just using its existence to tell if they were in review mode or not. Then if you looked at how `reviewModeRubric` was set, it kind of seemed like needless misdirection:

``` TSX
        // renderer.tsx checks if we're in `reviewMode` then sets `reviewModeRubric`
        const reviewModeRubric =
            this.props.reviewMode && widgetInfo ? widgetInfo.options : null;

        // passage.tsx for example then converted that object into a bool
        const editable = !this.props.reviewModeRubric;
```

So the crux of this PR is: why don't we just use `reviewMode` everywhere that's using coercing `reviewModeRubric` to a flag so that it's clearer what's actually using `reviewModeRubric` when we start work on LEMS-2437.